### PR TITLE
Add Ravelin Article

### PR DESCRIPTION
**Pull request by zrosenbauer**
# Description

Good article. Honest feedback about BigTable. I think it really gives a good look behind the curtains.

### CODE REVIEW

**Comment by zrosenbauer** [comment]
@sergiopalooza thanks for finding this BTW!

**Review by sergiopalooza — DISMISSED** [review]

**Review by sergiopalooza — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ A curated list of resources to help you get off the ground with Bigtable.
 - [BigTable: storing Protobuf bytes in one column vs splitting the content into column families/qualifiers](https://tech.travelaudience.com/bigtable-storing-protobuf-bytes-in-one-column-vs-splitting-the-content-into-column-families-c231bdff8db7)
 - [Cassandra: Daughter of Dynamo and BigTable](https://blog.insightdatascience.com/cassandra-daughter-of-dynamo-and-bigtable-1b57b16229b9)
 - [Using Google Cloud Emulators in Integration Tests](https://medium.com/google-cloud/using-google-cloud-emulators-for-integration-tests-7812890ebe0d)
+- [The Joy and Pain of using Google BigTable](https://syslog.ravelin.com/the-joy-and-pain-of-using-google-bigtable-4210604c75be)
```
Comment (mbrukman) [inline README.md]: I realize this is replicating the article, but could you please capitalize it as "Bigtable" (lowercase 't')? I'll ask the authors to fix the title in their article, and I have PR #3 for fixing all existing instances of "BigTable" -> "Bigtable".

Comment (zrosenbauer) [reply README.md]: Let's go ahead and change this here and you work on getting the Blog title updated.

**Review by mbrukman — APPROVED** [review]
LGTM

**Pull request merged**